lib/ansiblelint/formatters/__init__.py

```diff
@@ -90,6 +91,42 @@ def format(self, match, colored=False):
                                     match.message)
 
 
+class AnnotationsFormatter(BaseFormatter):
+    # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message
+    """Emits errors in format that is recognized by Github Annotation parsing.
```

Comment:
This mustn't start with a verb because classes represent objects and don't do anything by themselves, the actions are in methods and methods should have imperative strings.

```suggestion
    """Formatter for emitting violations as GitHub Workflow Commands.

    These commands trigger the GHA Workflow runners platform to post violations
    in a form of GitHub Checks API annotations that appear rendered in pull-
    request files view.
```